fixture/filtering/filetype_with_comments.yaml

```diff
@@ -0,0 +1,4 @@
+filteringConfiguration:
```

Comment:
where the comments?

Reply:
This is a regular YAML comment which should work out of the box, added